src/librustc_llvm/build.rs

```diff
@@ -96,11 +83,11 @@ fn main() {
     let version_output = output(&mut version_cmd);
     let mut parts = version_output.split('.').take(2)
         .filter_map(|s| s.parse::<u32>().ok());
-    let (major, minor) =
+    let (major, _minor) =
         if let (Some(major), Some(minor)) = (parts.next(), parts.next()) {
             (major, minor)
```

Comment:
Shouldn't this `panic!` if it encounters a version < 3.9?

Reply:
We certainly could, but this is also [checked in bootstrap](https://github.com/rust-lang/rust/blob/bb42071f63830a984c4983f6fbdf982916857f72/src/bootstrap/native.rs#L266).